economy/tests/tests.py

```diff
@@ -115,3 +121,269 @@ def test__long_comment_value__renders_ellipses(self):
         self.deposit_comment.save()
 
         self.assertIn("..", str(self.deposit_comment))
+
+
+class DepositFormTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.deposit = DepositFactory()
+
+    def test__valid_deposit_form__is_valid_returns_true(self):
+        form = DepositForm(data={'amount': self.deposit.amount, 'description': self.deposit.description,
+                                 'receipt': self.deposit.receipt})
+        self.assertTrue(form.is_valid())
+
+    def test__invalid_deposit_form__is_valid_returns_false(self):
+        form = DepositForm(data={'description': self.deposit.description,
+                                 'receipt': self.deposit.receipt})
+        self.assertFalse(form.is_valid())
+
+
+class DepositCommentFormTest(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.deposit_comment = DepositCommentFactory()
+
+    def test__valid_deposit_comment_form__is_valid_returns_true(self):
+        form = DepositCommentForm(data={'comment': self.deposit_comment.comment})
+        self.assertTrue(form.is_valid())
+
+    def test__invalid_deposit_comment_form__is_valid_returns_false(self):
```

Comment:
Did you mean this instead?
```suggestion
    def test__invalid_deposit_comment_form__is_empty_returns_false(self):
```

Reply:
Semantics, its an invalid form and the test is asserting if is_valid returns false although I agree with the ambiguous nature of the test name. Will rename it.